stdlib/bytearray.mini

```diff
@@ -129,165 +126,139 @@ public func bytearray_size(ba: ByteArray) -> uint {
     return ba.size;
 }
 
+public func bytearray_buffer(ba: ByteArray) -> buffer {
+    return ba.contents;
+}
+
+public func bytearray_slice(ba: ByteArray) -> uint {
+    return ba.sliceOffset;
+}
+
 // bytearray_getByte reads one byte from a ByteArray
 public func bytearray_getByte(ba: ByteArray, offset: uint) -> uint {
     if (offset >= ba.size) {
         return 0;
     }
     offset = offset + ba.sliceOffset;
-    let word = expandingIntArray_get(ba.contents, offset/32);
-    return asm(offset & 0x1f, word) uint { byte };
+    let res = getbuffer8(ba.contents, offset);
+    return res;
```

Comment:
This would be better as a direct return since I don't think the compiler optimizes this